lib/timetable-form/styles.module.css

```diff
@@ -1,3 +1,12 @@
 .form {
   padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px
+}
+
+.formRow {
```

Comment:
This is still camel case?

Reply:
My bad. I forgot to use kebab-case after changing the class name. Done now 🙂